Source/WebKit/Shared/ios/InteractionInformationAtPosition.h

```diff
@@ -85,6 +85,7 @@ struct InteractionInformationAtPosition {
     bool shouldNotUseIBeamInEditableContent { false };
     bool isImageOverlayText { false };
     bool isVerticalWritingMode { false };
+    bool isUsingAlternateURLForImage { false };
```

Comment:
@whsieh Any thoughts on the new bit here to support this functionality?

(@rr-codes you will also need to update the encoder/decoder in `InteractionInformationAtPosition.m` if we go with this solution).

Reply:
Hm...so I think it _is_ a bit weird, since it would be the only piece of information on the position information that's not derived from web-process-side state.

Reply:
Another alternative is to simply introduce a new flag on `WKContentView`, similar to `_contextMenuActionProviderDelegateNeedsOverride`, and then consult that later on. You can also follow the same lifecycle as `_contextMenuActionProviderDelegateNeedsOverride`, in terms of when to reset the flag to `NO`.

Reply:
@whsieh so should I do that instead?

Reply:
I would prefer that, over a bit hanging off of position information. But I'm curious if @pxlcoder or @hortont424 have opinions on this too!

Reply:
We could also make it derived from web process state by negating the condition, and setting to `true` in the web process when the `url` is initially null. It is already a little strange that we currently overwrite `_positionInformation.url` in the UI process, but changing that behavior could have unexpected consequences.

Though the flag sounds fine to me, since we avoid unnecessarily increasing the size of position information.

Reply:
+1!

I suppose, I have the same thoughts regarding `_positionInformation.url` as well (I think it would be cleaner to move that out of position information, and into state on the content view). But that is surely a change that should happen as a separate change.